public/.well-known/apple-app-site-association.json

```diff
@@ -4,7 +4,7 @@
     "details": [
       {
         "appID": "DQ48D9BF2V.org.cru.mpdx",
-        "paths": ["/contacts/*", "/auth/mobile"]
+        "paths": ["/accountLists/*/contacts/*", "/auth/mobile"]
```

Comment:
@levieggertcru I'm assuming including a wildcard in the middle of the apple-app-site-association path would work correctly for iOS?

Reply:
When researching, this is what I found I should do when it's a dynamic route. But I could be wrong.

Reply:
I just checked the Universal Links [spec](https://developer.apple.com/library/archive/documentation/General/Conceptual/AppSearch/UniversalLinks.html) and the wildcard pattern is valid according to the spec.